workers/toolpath/worker/toolpathworker.js

```diff
@@ -7,7 +7,7 @@ if (typeof window == "undefined") { // Only run as worker
 
   self.addEventListener('message', function(e) {
     // console.log("New message received by worker", e.data.data.length)
-    importScripts("/lib/clipperjs/clipper_unminified.js", "/lib/threejs/three.min.js", "/lib/gl-matrix.js", "/lib/tbfleming/web-cam-cpp.js");
```

Comment:
I guess this root relative paths aren't working in Electron?

Reply:
Unfortunately no

Reply:
That's surprising - it most be foldered differently than I expected. The relative paths you changed them to should be fine then.